P/pocl/pocl@7/common.jl

```diff
@@ -108,9 +108,20 @@ function build_script(standalone=false)
         CMAKE_FLAGS+=(-DLLVM_SPIRV_LIB="${prefix}/bin/libLLVMSPIRVLib.dll")
     fi
     ## spoof the output of the try_run test (which fails in cross-compilation mode)
+    MAX_SPIRV_VERSION=67072
```

Comment:
This comes from the headers (reinterpreted as integer), https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/590068b12402262dd9bd0aa1386f2e89babfa8a1/include/LLVMSPIRVOpts.h#L57-L70, but I figured its simpler to hard-code instead of trying to parse that out of it.